packages/@aws-cdk/aws-iam/package.json

```diff
@@ -84,6 +84,7 @@
   },
   "awslint": {
     "exclude": [
+      "from-signature:@aws-cdk/aws-iam.Role.fromRoleArn",
```

Comment:
Depending on how "done" you are with this change... it might actually be worth updating the linter rule to accept the additional options argument. I'm okay with you skipping that if you don't feel like it right now.

Reply:
I tried, but it's actually a non-trivial amount of work to add a concept of an optional method parameter to the current linter, so I've parked this for now.